webapp/src/ts/app.component.ts

```diff
@@ -364,6 +368,18 @@ export class AppComponent implements OnInit {
     });
   }
 
+  private watchDatabaseConnection() {
+    this.databaseConnectionMonitorService
+      .listenForDatabaseClosed()
+      .subscribe(() => {
+        this.modalService
+          .show(DatabaseClosedComponent)
+          .catch(() => {});
+
+        // ToDo: closeDropdowns();
```

Comment:
This closeDropdowns() hasn't been migrated yet. Part of the app component migration ticket.